assets/js/components/settings/SettingsAdmin.js

```diff
@@ -68,9 +67,9 @@ const SettingsAdmin = () => {
 	}, [ isUserInputCompleted ] );
 
 	return (
-		<Fragment>
+		<div className="mdc-layout-grid__inner">
```

Comment:
This looks like it should be a `Row` instead 🙂

Reply:
Nice catch, thanks 👍🏻